deploy/k8s/charts/trustification/templates/services/report/010-Secret.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if and .Values.modules.report.enabled .Values.modules.report.enabledAuth}}
+kind: Secret
+apiVersion: v1
+metadata:
+  name: report-auth-secret
+  labels:
+    app.kubernetes.io/name: report
+    app.kubernetes.io/component: trustification
+    app.kubernetes.io/part-of: trustification
+data:
+  .htpasswd: "CmFkbWluOiRhcHIxJFhGQ0ZmenU0JHZ0Zm94MXFlQlBxY1p2ODE0a2x0TDEK"
```

Comment:
There must not be a hard-coded password.

Reply:
this is the content of .htpasswd.  I do not understand how to "no hard-coded".

Reply:
Can the user change this content through the Helm chart?

Reply:
the content from cat .htpasswd. so use can change it.